### ISSUE TITLE
Pie chart with only one item produces empty circle

### ISSUE THREAD

**Issue by tarkatronic** [DmitryBaranovskiy/g.raphael#3]
Given the following:

```
<table><tr><th>Item 1</th><td>5</td></tr></table>
<div id="holder"></div>
<script type="text/javascript">
(function(raphael) {
    $(function(){
        values = labels = [];
        $("tr").each(function(){
            values.push(parseInt($("td",this).text()));
            labels.push($("th",this).text());
        }
        raphael("holder",840,500).g.piechart(220,220,200,values,,{legend:labels,legendpos:'east'});
    });
})(Raphael.ninja());
```

All I see is an empty circle, with a single pixel black border. I am running into this situation, because I am using the pie chart for reporting on live results of a poll on my website, and so far everybody is giving the same answer.

**Comment by coin** [comment]
Indeed; I just experienced a similar bug, I'm using the pie chart to give newsletters statistics about how many time the links are clicked, but in my case I don't even have the black border, the pie just vanish.

**Comment by nelsou** [comment]
FIX:

On line 23 of http://github.com/DmitryBaranovskiy/g.raphael/blob/master/g.pie.js
replace "stroke:opt.stroke" by "stroke:opts.stroke"

Should be better :)

**Comment by rokcarl** [comment]
I have the same problem and this fix does not fix it for me, still the same 1px black border.

**Comment by impressiver** [comment]
in the minimized js file, replace "stroke:opt.stroke" with "stroke:o.stroke"

**Comment by aivijay** [comment]
I have the same problem with one item. With the above changes nothing happens. 
On firefox nothing shows up.
On Chrome the filled circle shows up, on hover it vanishes. 
Any ideas why this is happening?

**Comment by aivijay** [comment]
I resorted finally to do the changes as above in g.pie.js setting it to stroke:o.stroke though it didnt fix the problem. I set one of the other data as well to a fraction 0.001 and was able to get the chart shown as needed with 100% for the one which needs to show up but there is a very fraction which doesnt effect the chart for now. 

Maybe this might help someone having a similar problem.

<a href="http://www.mailtics.com">Mailtics.com</a>
-vi

**Comment by aivijay** [comment]
you can check it out at <a href="http://mailtics.com">mailtics.com</a>

**Comment by fredpyo** [comment]
The problem is that according to w3c's SVG spec (http://www.w3.org/TR/SVG/implnote.html#ArcOutOfRangeParameters) if the start and endpoint match, then no arc is drawn.

**Comment by frostbytten** [comment]
To anyone who needs to just put a dot (100% of a value for example):

The following commit solves this problem, as well as adding a couple of features for the pie chart. My group is extending this library for our cross-platform graphing library.

open-agroclimate@3fe60f36c9c2ddb56b25d11ee91775447a3715c1

Best of luck
